google_compute_engine/metadata_scripts/tests/script_retriever_test.py

```diff
@@ -34,6 +34,15 @@ def setUp(self):
         self.mock_logger, self.script_type)
 
   @mock.patch('google_compute_engine.metadata_scripts.script_retriever.subprocess.check_call')
+  def testDownloadGsNoExec(self, mock_call):
```

Comment:
I think that the change (running `which gsutil`) dropped test coverage in testDownloadGsUrlProcessError. Please modify mock_call.side_effect (line 65) so that it returns success for gsutil.